lisk-dex-electron/src/PlaceOrder.jsx

```diff
@@ -318,6 +329,23 @@ export default class PlaceOrder extends React.Component {
     }
   }
 
+  getAssetBalance() {
+    const whole = 10 ** 8;
+    return Math.round((this.props.assetBalance * 100) / whole) / 100;
```

Comment:
@jondubois Also about the getAssetBalance function, isn't 10 ** 8 also supposed to come from config.json?

Reply:
Ah yes it is. I missed that one.

Reply:
Btw. why do we multiply by 100 and then divide by 100 again